### DISCUSSION TITLE
2022-04-08 Data Platform Team Updates

### DISCUSSION THREAD

**Discussion by khorolets** [near/community#242] [Archived]
👋 Greetings! Data Platform Team is here with the updates. Don't have much to say, especially in comparison with the previous exciting week 😅. We are mostly focused on rewriting the Indexer for Explorer. Meanwhile, we are working on the creation of a whole new space with indexer docs, we want to create a knowledge base to share with everyone to help understand what indexer is, how to create one, and find out whether you need it in the first place.

Some details:

* Continue working on [the new version of Indexer](https://github.com/telezhnaya/near-lake-flows-into-sql-base). While we are still in the process of choosing the libraries, stress testing is started. Btw, if you want to change the naming, it's the right time to participate in the discussion [here](https://github.com/telezhnaya/near-lake-flows-into-sql-base/issues/6) (@telezhnaya)
* Working on setting up the indexer docs home (@khorolets)
* We've written a big article explaining the entire concept of indexers, described indexer projects we are running (@khorolets)
* Almost finished with a new video explaining how the Data flows in NEAR Blockchain (@khorolets)

We're planning to release **Indexer Docs next week**, so stay tuned it should be useful and fun.

Anyway, the week was pretty productive. Although, we believe next week will be even more exciting. Stay with us, keep on rolling 🤘. See you all next week!

**Comment by idea404** [comment]
Hey @khorolets, where will you guys share a link to that video for data flow through NEAR blockchain? I'm getting started on a visual outlining the network architecture and this would likely be very helpful!

**Reply by khorolets** [reply]
@idea404 we're going to publish it on YouTube this week. I can ping you here once it is pusblished.
